src/quantum/index.jsx

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import { parse } from 'query-string';
 import Grid from '@material-ui/core/Grid/Grid';
 import DashboardPage from '../components/DashboardPage';
-import Perfherder from './perfherder';
+import PerfherderWidget from './perfherder';
```

Comment:
In the future, don't spend your time investing on this component.
We need to switch these two graphs to using `PerfherderGraphContainer` and we can just kill the old code:
<img width="1039" alt="image" src="https://user-images.githubusercontent.com/44410/54162996-f2d87580-442d-11e9-960f-60d058cc8e3a.png">